src/widgets/FormDateInput.js

```diff
@@ -111,6 +111,7 @@ export const FormDateInput = React.forwardRef(
               mode="date"
               display="spinner"
               value={pickerSeedValue}
+              maximumDate={new Date()}
```

Comment:
NIT: this is going to run the constructor on every render of the `DateTimePicker`. Unlikely that the day will tick over while the user is editing, so think a good compromise is to recalculate the max date when the date picker is closed/opened.

 `const maximumDate = useMemo(() => new Date(), [datePickerOpen]);`

Reply:
On top of that would cause a re-render on `DateTimePicker` as it will be a new reference (I'm assuming `DateTimePicker` is memoized) - but technically none of this occurs if it's not open - and if it's open then it won't be re-rendering from `FormDateInput`! 

So the memoization would do nothing (it will run the constructor and render once per update of `datePickerOpen`)